iwyu.cc

```diff
@@ -2669,6 +2673,9 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
     visitor_state_->processed_overload_locs.insert(loc);
   }
 
+  void ReportTemplateSpecTypeInternals(const TemplateSpecializationType*) {
```

Comment:
It's long, but I think `ReportTplSpecComponentTypes` matches terminology used in other parts of IWYU.